pkg/operator/resource/resourceapply/generic.go

```diff
@@ -170,12 +170,12 @@ func ApplyDirectly(clients *ClientHolder, recorder events.Recorder, manifests As
 			if clients.kubeClient == nil {
 				result.Error = fmt.Errorf("missing kubeClient")
 			}
-			result.Result, result.Changed, result.Error = ApplyValidatingWebhookConfiguration(clients.kubeClient.AdmissionregistrationV1(), recorder, t)
+			result.Result, result.Changed, result.Error = ApplyValidatingWebhookConfiguration(clients.kubeClient.AdmissionregistrationV1(), recorder, t, 0)
```

Comment:
Since they need expectedGeneration now, should we remove them from `ApplyDirectly`? It will keep overwriting the existing object and that's not the behavior we want to support.

Reply:
This behavior is equal to other `ApplyDirectly` methods https://github.com/openshift/library-go/blob/c8d0c55677d43fef621e8afb5a2c092ed75a5baf/pkg%2Foperator%2Fresource%2Fresourceapply%2Fstorage.go#L90 I added a test case which shows how it will work.

Reply:
`CSIDriver` does not have `Generation` and can't be (easily) compared, direct apply is the only way. Anyway, I am not going to use `ApplyDirectly` / StaticController, so feel free to ignore this comment.